imaginary.go

```diff
@@ -154,6 +157,11 @@ func main() {
 		checkHttpCacheTtl(*aHTTPCacheTTL)
 	}
 
+	// Parse endpoint names to disabled, if present
+	if *aDisableEndpoints != "" {
+		opts.Endpoints = parseEndpoints(*aDisableEndpoints)
```

Comment:
Should we name is `disableEndpoints`? Otherwise it may confuse since we parse disable endpoint but assign to `Endpoints` field

Reply:
For simplification I would prefer that field name.